couchbase/json-buckets.yml

```diff
@@ -0,0 +1,968 @@
+        type: long_gauge
+        label: delete misses
+        description: Delete misses
+
+      - name: disk.commit.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.disk_commit_count[0]
+        type: long_gauge
+        label: disk commits
+        description: Disk commits
+
+      - name: disk.update.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.disk_update_count[0]
+        type: long_gauge
+        label: disk updates
+        description: Disk updates
+
+      - name: disk.write.queue
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.disk_write_queue[0]
+        type: long_gauge
+        label: disk writes
+        description: Disk write queue depth
+
+      - name: ep.bg.fetched
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_bg_fetched[0]
+        type: long_gauge
+        label: disk reads
+        description: Disk reads per second
+
+      - name: ep.cache.miss.rate
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_cache_miss_rate[0]
+        type: long_gauge
+        label: cache miss rate
+        description: Cache miss rate
+
+      - name: ep.cache.miss.ratio
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_cache_miss_ratio[0]
+        type: long_gauge
+        label: cach miss ratio
+        unit: "%"
+        description: Cache miss ratio
+
+      - name: ep.dcp.fts.backoff
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_fts_backoff[0]
+        type: long_gauge
+        label: backoffs for fts DCP
+        description: Number of backoffs for fts DCP connections
+
+      - name: ep.dcp.fts.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_fts_count[0]
+        type: long_gauge
+        label: fts DCP
+        description: Number of fts DCP connections
+
+      - name: ep.dcp.fts.items.remaining
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_fts_items_remaining[0]
+        type: long_gauge
+        label: remaining fts item
+        description: Number of fts items remaining to be sent
+
+      - name: ep.dcp.fts.items.sent
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_fts_items_sent[0]
+        type: long_gauge
+        label: sent fts items
+        description: Number of fts items sent
+
+      - name: ep.dcp.fts.producer.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_fts_producer_count[0]
+        type: long_gauge
+        label: fts producer
+        description: Number of fts producers
+
+      - name: ep.dcp.2i.total.bytes
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_2i_total_bytes[0]
+        type: long_gauge
+        label: sent indexes DCP
+        unit: bytes/second
```

Comment:
Doesn't match metric name. If it is really bytes/second and a gauge it will not be useful for us, please double check. Same comment for other metrics below.

Reply:
The description is "Number bytes per second being sent for indexes DCP connections"

Reply:
I change to doubel_gauge

Reply:
Yes, description seem not match but that maybe just the metric name is not well, description is correct.
I found description about this metric in couchbase code

{desc, <<"Number of bytes per second being sent for views/indexes DCP connections "
                            "for this bucket (measured from ep_dcp_views_total_bytes + "
                            "ep_dcp_2i_total_bytes + ep_dcp_fts_total_bytes)">>}]},